pupil_src/launchables/args.py

```diff
@@ -0,0 +1,93 @@
+"""
+(*)~---------------------------------------------------------------------------
+Pupil - eye tracking platform
+Copyright (C) 2012-2018 Pupil Labs
+
+Distributed under the terms of the GNU
+Lesser General Public License (LGPL v3.0).
+See COPYING and COPYING.LESSER for license details.
+---------------------------------------------------------------------------~(*)
+"""
+
+import argparse
+import sys
+
+
+def parse(*, running_from_bundle, app="capture", port=None, **kwargs):
```

Comment:
By specifying `*`, `running_from_bundle` is not allowed to be passed as positional argument anymore. It is a keyword-only argument without a default value. This way I prohibit users to call `parse(boolean)` and force them to pass the keyword: `parse(running_from_bundle=boolean)`.

The app+port defaults are needed to be named explicitly for the caveat below.

Reply:
What's the reason to be so strict about not passing booleans directly here? I think that's something that should rather be enforced by reviews. For example, now you need to write `running_from_bundle=running_from_bundle` which is superfluous.

Would it be possible to do something like the following?
```
if "app" not in kwargs:
    kwargs["app"] = "capture"
if "port" not in kwargs:
    kwargs["port"] = None
```
imo, this makes it clearer that you are setting defaults in case its not included in `kwargs`

Reply:
I came up with an alternative implementation in 528d0b683f0134540e030d9d9627fd22785749ef